string label
float32 var_x
float32 var_y
